staging/src/k8s.io/kubectl/pkg/explain/v2/generator_test.go

```diff
@@ -30,7 +31,16 @@ func TestGeneratorMissingOutput(t *testing.T) {
 	var buf bytes.Buffer
 	var doc map[string]interface{}
 
-	err := json.Unmarshal([]byte(apiDiscoveryJSON), &doc)
+	fakeClient, err := openapitest.NewFileOpenAPIClient("api/openapi-spec/v3")
+	require.NoError(t, err)
+	paths, err := fakeClient.Paths()
+	require.NoError(t, err)
+	gv, found := paths["apis/discovery.k8s.io/v1"]
+	require.True(t, found)
+	discoveryBytes, err := gv.Schema("unused")
+	require.NoError(t, err)
```

Comment:
I dont think it makes sense to put this code for testing the fake discovery client into every explain test

Reply:
Yeah, we should keep the test simple and focused

Reply:
@alexzielenski I've updated this; please let me know what you think.